src/actors/resolver.rs

```diff
@@ -3,6 +3,7 @@
 //! ## Example
 //!
 //! ```rust
+//! use futures_util::{future, future::FutureExt};
```

Comment:
I think this import is not necessary for this example. Probably it's here only due to a rebase as a similar line was been removed recently.

Reply:
I have removed this. :+1: